marimo/_messaging/ops.py

```diff
@@ -248,12 +254,32 @@ def broadcast_error(
         cell_id: CellId_t,
     ) -> None:
         console: Optional[list[CellOutput]] = [] if clear_console else None
+
+        # In run mode, we don't want to broadcast the error. Instead we want to print the error to the console
+        # and then broadcast a new error such that the data is hidden.
+        safe_errors: list[Error] = []
```

Comment:
I can imagine a use case for which broadcasting still makes sense. But I guess this is okay as a default, and we can make it configurable in the future if people ask.